sdk/python/setup.py

```diff
@@ -43,7 +43,7 @@
     'kfp-server-api>=1.1.2,<2.0.0',
     'jsonschema>=3.0.1,<4',
     'tabulate>=0.8.6,<1',
-    'click>=7.1.1,<8',
+    'click>=7.1.2,<9',
```

Comment:
Since this is a major version bump, have we checked if there was any breaking change from 7 to 8? I'm afraid we don't have test coverage around the SDK CLI, so you probably have to read their changelog and check our code usage on it.